src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write your code here
+const tableRows = document.querySelectorAll('tr');
+
+tableRows.forEach(row => {
+  row.children[row.children.length - 1].before(row.children[1].cloneNode(true));
```

Comment:
agree with @TheVovchik 
this code is hard to read, don't write everything in one line, create variables, make your code more readable